ONSdigital/eq-author
# 459 update toggle and numeric field styles

### What is the context of this PR?
- https://trello.com/c/FcPqON0I/459-update-toggle-switch-and-numeric-field-to-match-new-designs
- Updated toggle component to match new designs
- Removed increment/decrement buttons from number components

### How to review 
- Run tests
- See visual changes in Storybook
